dev/cloud/toc.yml

```
@@ -16,4 +16,6 @@
   - name: Automapper
     href: automapper.md
   - name: Repositories
-    href: repositories.md
\ No newline at end of file
+    href: repositories.md
+  - name: Naming Conventions
+    href: naming_conventions.md
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)